pkg/pods/pod.go

```diff
@@ -602,7 +603,11 @@ func (pod *Pod) setupConfig(manifest manifest.Manifest, launchables []launch.Lau
 		if err != nil {
 			return err
 		}
-		err = writeEnvFile(launchable.EnvDir(), "LAUNCHABLE_ROOT", launchable.InstallDir(), uid, gid)
+		err = writeEnvFile(launchable.EnvDir(), LaunchableRootEnvVar, launchable.InstallDir(), uid, gid)
+		if err != nil {
+			return err
+		}
+		err = writeEnvFile(launchable.EnvDir(), LaunchableRestartTimeoutEnvVar, fmt.Sprintf("%s", launchable.GetRestartTimeout()*time.Second), uid, gid)
```

Comment:
have you checked how this turns into a string? seems like it's already a `time.Duration` so you probably don't need to `*time.Second`

Reply:
fair cop. Here's what go thinks: https://play.golang.org/p/wJQzqbhN-T

I like the explicit int64() conversion - WDYT?